### PR TITLE
Simple icon redirect

### PR DESCRIPTION
**Pull request by anthonyoliai**
Had someone comment , especially in mobile that they wanted to be able to get redirected by tapping on the 1Hive icon. Quickly made a redirect, not sure if there is a cleaner way of doing this :)

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -8,7 +8,13 @@ import headerBackgroundSvg from '../assets/header-background.svg'
 import logoSvg from '../assets/logotext.svg'
 
 function Header({ compact }) {
-  const BeeIcon = <img src={beeSvg} height={compact ? 40 : 60} alt="" />
+  const BeeIcon = (
+    <a href="https://1hive.org">
+      {' '}
```
Comment (rperez89) [inline src/components/Header.js:13]: i think we can remove this space right?

**Review by rperez89 — COMMENTED** [review]
Thxs man, now that you are getting more involved i think that might be good if you can reach me or @fabriziovigevani before submitting a PR for a feature requested in the community because we are launching the v2 soon and you might end working on stuffs that might not be needed! just to coordinate!

**Comment by anthonyoliai** [comment]
> Thxs man, now that you are getting more involved i think that might be good if you can reach me or @fabriziovigevani before submitting a PR for a feature requested in the community because we are launching the v2 soon and you might end working on stuffs that might not be needed! just to coordinate!

Alright! Very sorry about that, I will coordinate more :) sorry still getting used to this work setting 😅

**Comment by anthonyoliai** [comment]
I had a talk earlier with @sembrestels today regarding honeypot v1, as the changes in v1 could be handy to v2 too. And we've come to the conclusions that there are some simple quality of life problems such as this that can be easily fixable. Hence the follow up PR, hope you can have a look.

File: src/components/Header.js [hunk 2]
```diff
@@ -8,7 +8,12 @@ import headerBackgroundSvg from '../assets/header-background.svg'
 import logoSvg from '../assets/logotext.svg'
 
 function Header({ compact }) {
-  const BeeIcon = <img src={beeSvg} height={compact ? 40 : 60} alt="" />
+  const BeeIcon = (
+    <a href="https://1hive.org">
```
Comment (fabriziovigevani) [inline src/components/Header.js:12]: We can use the [`Link`](https://ui.aragon.org/link/) component. from `@1hive-ui` in this case, we can also use relative path.

```suggestion
    <Link href="/" external={false}>
```

**Comment by fabriziovigevani** [comment]
I think as long as they are adding small features that bring value to the current version it's fine. And i agree they can also be used for V2. But yeah when is a feature or change more involved then let's discuss if it's worth doing it for this version or v2 :)

File: src/components/Header.js (hunk 2 above)
Comment (anthonyoliai) [reply src/components/Header.js:12]: That's a much cleaner way of doing it! Shows how inexperienced I am with React as of now haha. Putting up a new PR, thanks!

File: src/components/Header.js [hunk 3]
```diff
@@ -9,9 +9,9 @@ import logoSvg from '../assets/logotext.svg'
 
 function Header({ compact }) {
   const BeeIcon = (
-    <a href="https://1hive.org">
+    <Link href="/" external={false}>
```
Comment (fabriziovigevani) [inline src/components/Header.js:12]: Ah sorry, forgot we are using a hash router, lets add the `#`

```suggestion
    <Link href="/#" external={false}>
```

**Pull request merged**